test/JDBC/expected/restricting_permission_on_role.out

```diff
@@ -0,0 +1,167 @@
+-- tsql
+create login restricting_permission_on_role_l1 with password = '123';
```

Comment:
Let's have `tsql_login` in the above login's name and `psql_user` in psql user below for better clarity.